### PR TITLE
Using Channels for Realtime is incredibly Easy! 

### PR DESCRIPTION
**Pull request by nelsonic**
Adds channels code snapshots from the Programming Phoenix book in case anyone `else` is following along and gets stuck with any parts of the code.

Not tied to a _specific_ issue. but linked to https://github.com/dwyl/learn-phoenix-framework/issues/2 👍

### CODE REVIEW

File: rumbl/web/controllers/auth.ex [hunk 1]
```diff
@@ -8,22 +8,33 @@ defmodule Rumbl.Auth do
 
   def call(conn, repo) do
     user_id = get_session(conn, :user_id)
+
     cond do
       user = conn.assigns[:current_user] -> conn
+        put_current_user(conn, user)
```
Comment (shouston3) [inline rumbl/web/controllers/auth.ex]: I'm not sure if I'm just reading this wrong, but don't you need to remove the `conn` at the end of
`user = conn.assigns[:current_user] -> conn` line (or perhaps pipe it in to the `put_current_user` function or something.

Hard to say, I'm not this far in the book yet

Comment (nelsonic) [inline rumbl/web/controllers/auth.ex]: @shouston3 both would be valid. This is one fewer line and achieves the objective.
As soon as we need to pipe more than one function we would convert it. 👍

**Comment by nelsonic** [comment]
@shouston3 I don't have an "_opinion_" on the use of ES6.
I _know_ that ES6 does not work in sizeable portion of browsers.
I'm merely copying out the code in the Examples in the book.
When I'm done with the book I will re-write the app in "PETE" and it will be _glorious_. 😉

File: rumbl/web/controllers/auth.ex (hunk 1 above)
Comment (shouston3) [inline rumbl/web/controllers/auth.ex]: I'm not saying to pipe it.

I'm saying the `conn` on the end of line 12/13 is not doing anything.

**Pull request merged**